package.json

```diff
@@ -1,1637 +1,1748 @@
+                },
+                "mssql.query.ansiDefaults": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "%mssql.query.ansiDefaults%"
+                },
+                "mssql.query.quotedIdentifier": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "%mssql.query.quotedIdentifier%"
+                },
+                "mssql.query.ansiNullDefaultOn": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "%mssql.query.ansiNullDefaultOn%"
+                },
+                "mssql.query.implicitTransactions": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "%mssql.query.implicitTransactions%"
+                },
+                "mssql.query.cursorCloseOnCommit": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "%mssql.query.cursorCloseOnCommit%"
+                },
+                "mssql.query.ansiPadding": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "%mssql.query.ansiPadding%"
+                },
+                "mssql.query.ansiWarnings": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "%mssql.query.ansiWarnings%"
+                },
+                "mssql.query.ansiNulls": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "%mssql.query.ansiNulls%"
+                },
+                "mssql.query.alwaysEncryptedParameterization": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "%mssql.query.alwaysEncryptedParameterization%"
+                },
+                "mssql.ignorePlatformWarning": {
+                    "type": "boolean",
+                    "description": "%mssql.ignorePlatformWarning%",
+                    "default": false
+                },
+                "mssql.objectExplorer.groupBySchema": {
+                    "type": "boolean",
+                    "description": "%mssql.objectExplorer.groupBySchema%",
+                    "default": false
+                },
+                "mssql.objectExplorer.expandTimeout": {
+                    "type": "number",
+                    "default": 45,
+                    "minimum": 1,
+                    "description": "%mssql.objectExplorer.expandTimeout%"
+                }
+            }
         },
-        "mssql.objectExplorer.expandTimeout": {
-          "type": "number",
-          "default": 45,
-          "minimum": 1,
-          "description": "%mssql.objectExplorer.expandTimeout%"
-        }
-      }
+        "walkthroughs": [
```

Comment:
Besides for this walkthroughs section, are the rest of the changes formatting changes?

Reply:
Oi, yes.  Lemme remove these formatting changes.  They shouldn't be in this PR.